reverbePrintemps/spiced
# Complete challenge

# Context

This PR aims to tackle [the filter method challenge](https://github.com/spiced-academy/honey-web-dev/blob/main/sessions/js-array-methods/challenges-js-array-methods.md#filter).

# Changes

- [x] Assigned `onlyCardWithIdTwo` to the card with the id `2`
- [x] Assigned `allCardsWith3Tags` to all cards that have 3 tags
- [x] Assigned `allCardsThatAreNotBookmarked` to all cards that are not bookmarked
- [x] Assigned `allCardsWithTagsHTMLOrJSThatAreBookmarked` to all cards that are bookmarked and have the tag `html` or `js`

# How to test

1. Clone repo
2. Checkout this branch
3. Run `npm run test`
4. Verify that all tests pass

# TODO


# GIF

![](https://media.giphy.com/media/wGij09pD572aucJJHj/giphy.gif)
